test/benchee/benchmark_test.exs

```diff
@@ -52,9 +52,11 @@ defmodule Benchee.BenchmarkTest do
 
         assert new_suite.configuration == suite.configuration
         run_times_hash = new_suite.run_times |> no_input_access
+        memory_usage_hash = new_suite.memory_usages |> no_input_access
 
         # should be 6 but gotta give it a bit leeway
         assert length(run_times_hash["Name"]) >= 5
+        assert length(memory_usage_hash["Name"]) >= 5
```

Comment:
I think the overhead is big enough that these values don't seem to hold anymore (see travis)